apps/financial-aid/web-veita/src/components/Profile/EmployeeProfile.tsx

```diff
@@ -35,7 +35,7 @@ interface EmployeeProfileInfo {
   roles: StaffRole[]
 }
 
-type InputType = 'text' | 'number' | 'email' | 'tel'
+export type InputType = 'text' | 'number' | 'email' | 'tel'
```

Comment:
Since this is now exported and probably used in other places than this component then we should move this to `types.ts`